packages/react/src/lib/felaRenderer.tsx

```diff
@@ -62,21 +65,15 @@ const createRendererConfig = (options: any = {}) => ({
     // Heads up!
     // This is required after fela-plugin-prefixer to resolve the array of fallback values prefixer produces.
     felaPluginFallbackValue(),
+
```

Comment:
`felaExpandCssShorthandsPlugin` should go after `felaPluginFallbackValue`. Watch out for the next merge with master.

Reply:
Merged this update